component-library/components/generic/custom-icon/custom-icon.bookshop.yml

```
@@ -1,26 +1,28 @@
 # Metadata about this component, to be used in the CMS
 spec:
   structures:
+    - content_blocks
   label: "Custom Icon"
-  description: A icon (from https://heroicons.com/) with an optional border and customisable color
+  description: A icon (that you upload) with an optional border and customisable color
   icon: bolt
   tags: []
 
 # Defines the structure of this component, as well as the default values
 blueprint:
-  svg_icon: academic-cap
+  icon_path: 
   icon_size: small
   icon_type: solid
   rounded_border: false
   color: "#f9f9fb"
```

```suggestion
  color: "#7d7f7c"
```
The default white ish on a white background is really hard to see. Could we change this to a grey that is visible ish on both dark/light backgrounds ?